solar_installation/data/ir_model_fields.xml

```diff
@@ -0,0 +1,178 @@
+        <field name="field_description">Task </field>
+        <field name="model_id" ref="industry_fsm_report.x_project_task_worksheet_template_1_ir_model_1"/>
+        <field name="relation">project.task</field>
+        <field name="ttype">many2one</field>
+    </record>
+    <record id="new_related_field_he_128" model="ir.model.fields">
+        <field name="name">x_related_field_128</field>
+        <field name="field_description">New Related Field </field>
+        <field name="model_id" ref="helpdesk.model_helpdesk_ticket"/>
+        <field name="related">lot_id.x_warranty_end_date</field>
+        <field name="ttype">date</field>
+    </record>
+    <record id="new_related_field_he_49" model="ir.model.fields">
+        <field name="name">x_related_field_49</field>
+        <field name="field_description">New Related Field</field>
+        <field name="model_id" ref="helpdesk.model_helpdesk_ticket"/>
+        <field name="related">product_id.removal_time</field>
+        <field name="ttype">integer</field>
+    </record>
+    <record id="new_related_field_he_339" model="ir.model.fields">
+        <field name="name">x_related_field_339</field>
+        <field name="field_description">New Related  Field </field>
+        <field name="model_id" ref="helpdesk.model_helpdesk_ticket"/>
+        <field name="related">lot_id.x_warranty_end_date</field>
+        <field name="ttype">date</field>
+    </record>
+    <record id="new_related_field_he_related_field" model="ir.model.fields">
+        <field name="name">x_related_field_7</field>
+        <field name="field_description">New Related Field  </field>
+        <field name="model_id" ref="helpdesk.model_helpdesk_ticket"/>
+        <field name="related">lot_id.expiration_date</field>
+        <field name="ttype">datetime</field>
+    </record>
+    <record id="new_related_field_he_related_field_2" model="ir.model.fields">
+        <field name="name">x_related_field_2</field>
+        <field name="field_description">New Related  Field</field>
+        <field name="model_id" ref="helpdesk.model_helpdesk_ticket"/>
+        <field name="related">lot_id.removal_date</field>
+        <field name="ttype">datetime</field>
+    </record>
+    <record id="x_related_field_warranty_end_date" model="ir.model.fields">
+        <field name="name">x_warranty_end_date</field>
+        <field name="field_description">Warranty End Date</field>
+        <field name="model_id" ref="helpdesk.model_helpdesk_ticket"/>
+        <field name="related">lot_id.x_warranty_date</field>
+        <field name="ttype">date</field>
+    </record>
+    <record id="new_related_field_he_related_field_1" model="ir.model.fields">
+        <field name="name">x_related_field_1</field>
+        <field name="field_description">New  Related Field</field>
+        <field name="model_id" ref="helpdesk.model_helpdesk_ticket"/>
+        <field name="related">partner_id.sale_order_ids.order_line.product_id</field>
```

Comment:
I don't know why there is `partner_id.sale_order_ids`.
Maybe it will raise error in future